### PR TITLE
Prometheus logging

### PR DESCRIPTION
**Pull request by LasseRaatz**
#minor

### CODE REVIEW

**Comment by LasseRaatz** [comment]
Custom api tests have been added to this PR to make merging with master easier!!

File: backend/src/custom-server.js [hunk 1]
```diff
@@ -152,13 +121,13 @@ app.post('/:username/follow', async (req, res) => {
 
     if (!currentUserId) return res.status(401).send({ error: 'currentUserId is missing' });
 
-    const follower = await db.user.findByPk(currentUserId);
+    const follower = await userRepository.getById(currentUserId);
 
-    const followed = await db.user.findOne({ where: { username } });
+    const followed = await userRepository.getByUsername(username);
```
Comment (DagBAndersen) [inline backend/src/custom-server.js]: Det hedder _followee_ 😎

File: backend/src/custom-server.js [hunk 2]
```diff
@@ -44,48 +45,20 @@ app.get('/public', async (req, res) => {
...
-    pubDate: msg.createdAt,
-    userId: msg.user.id,
-    username: msg.user.username,
-    email: msg.user.email,
-  })));
+  const user = await userRepository.getById(userId);
+
+  const messages = await messageRepository.getTimeline(user, numMessages);
+
+  return messages;
```
Comment (DagBAndersen) [inline backend/src/custom-server.js]: Flueknep, hvis du magter at ændre det... return await med det samme i stedet for at lav en lokalvariable

Comment (DagBAndersen) [inline backend/src/custom-server.js]:
```suggestion
return await messageRepository.getTimeline(user, numMessages);
```

File: backend/src/custom-server.js [hunk 3]
```diff
@@ -44,48 +45,18 @@ app.get('/public', async (req, res) => {
...
-  return (messages.map((msg) => ({
-    text: msg.text,
-    pubDate: msg.createdAt,
-    userId: msg.user.id,
-    username: msg.user.username,
-    email: msg.user.email,
-  })));
+  const user = await userRepository.getById(userId);
+
+return await messageRepository.getTimeline(user, numMessages);
```
Comment (LasseRaatz) [inline backend/src/custom-server.js]:
```suggestion
const messages = await messageRepository.getTimeline(user, numMessages);

return messages;
```

File: backend/src/custom-server.js [hunk 4]
```diff
@@ -44,48 +45,20 @@ app.get('/public', async (req, res) => {
...
-  return (messages.map((msg) => ({
-    text: msg.text,
-    pubDate: msg.createdAt,
-    userId: msg.user.id,
-    username: msg.user.username,
-    email: msg.user.email,
-  })));
+  const user = await userRepository.getById(userId);
+
+const messages = await messageRepository.getTimeline(user, numMessages);
```
Comment (LasseRaatz) [inline backend/src/custom-server.js]:
```suggestion
  const messages = await messageRepository.getTimeline(user, numMessages);
```

Comment (LasseRaatz) [inline backend/src/custom-server.js]:
```suggestion
  const messages = await messageRepository.getTimeline(user, numMessages);
```

File: backend/src/custom-server.js [hunk 5]
```diff
@@ -44,48 +45,20 @@ app.get('/public', async (req, res) => {
...
-    pubDate: msg.createdAt,
-    userId: msg.user.id,
-    username: msg.user.username,
-    email: msg.user.email,
-  })));
+  const user = await userRepository.getById(userId);
+
+  const messages = await messageRepository.getTimeline(user, numMessages);
+
+return messages;
```
Comment (LasseRaatz) [inline backend/src/custom-server.js]:
```suggestion
  return messages;
```

File: backend/src/custom-server.js (hunk 4 above)
Comment (DagBAndersen) [reply backend/src/custom-server.js]: skrald <3

**Pull request merged**